multi-agent/elements/tools/web_fetch/web_fetch.py

```
@@ -0,0 +1,34 @@
+
+    name: str = "web_fetch"
+    description: str = "Fetch a web page and return its content as markdown."
+    args_schema = WebFetchArgs
+
+    def run(self, **kwargs: Any) -> Dict[str, Any]:
+        args = WebFetchArgs(**kwargs)
+
+        try:
+            with httpx.Client(follow_redirects=True) as client:
+                response = client.get(args.url)
+        except Exception as exc:
```

Ruff (0.15.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)